### ISSUE TITLE
Make selecting custom coding standard possible

### ISSUE THREAD

**Issue by fqqdk** [beberlei/netbeans-php-enhancements#8]
Currently the standards bundled with phpcs seem to be hardcoded into the plugin's code. We use our custom coding standard where I work, and we would benefit greatly from more flexible configuration options regarding what coding standard to use.

Phpcs can be configured to run a custom standard with the --standard=<standard> option, and beside the names installed in the Standards directory of phpcs, it also accepts a path prefixes, like 'path/to/my/Standard', where 'path/to/my/Standard.php' is expected to hold a subclass of the PHP_CodeSniffer_Standard class.
- a text field should be displayed which could be used to enter a path-prefixed name to provide more flexibility
- additionally, if the plugin could scan phpcs's Standards directory for the standard names to display in the combo box, that would also be great

Also, thanks for your work, it is a very nice plugin :)

**Comment by meza** [comment]
I'd like that too!

**Comment by szaboferee** [comment]
It would be a very useful feature. The standards in the combo box could be parsed from the outpuf of "phpcs -i".

**Comment by Verber** [comment]
I think it will be very useful feature

**Comment by confiq** [comment]
I also agree that this would be useful ! ! ! I can't change standard to my needs :(

**Comment by FMaz008** [comment]
Yup, that would be nice !

**Comment by simonseyer** [comment]
Yes, I also agree, it would be a nice feature.

**Comment by HemaliSavaliya** [comment]
Will you please tell me that how to do this thing in net beans 8.1 because that same thing has been done in terminal but not in net beans
